codespell_lib/tests/test_basic.py

```diff
@@ -217,11 +217,17 @@ def test_ignore_dictionary(tmpdir, capsys):
     """Test ignore dictionary functionality."""
     d = str(tmpdir)
     with open(op.join(d, 'bad.txt'), 'w') as f:
-        f.write('1 abandonned 1\n2 abandonned 2\nabondon\n')
+        f.write('1 abandonned 1\n'
+                '2 abandonned 2\n'
+                '3 abilty 3\n'
+                '4 abilty 4\n'
+                '5 ackward 5\n'
+                '6 ackward 6\n'
+                'abondon\n')
```

Comment:
It would probably be good to mix in some \r\n in here too.
```suggestion
        f.write('1 abandonned 1\n'
                '2 abandonned 2\n'
                '3 abandonned 3\r\n'
                '4 abilty 4\n'
                '5 abilty 5\n'
                '6 abilty 6\r\n'
                '7 ackward 7\n'
                '8 ackward 8\n'
                '9 ackward 9\r\n'
                'abondon\n')
```